docs/content/latest/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -0,0 +1,138 @@
+---
```

Comment:
I still find it impossible to review PRs that just move stuff around...can you confirm this is basically the existing content, but with the 2-3 manual changes we discussed, eg: not mentioning replay

Reply:
yup :)